taxi/views.py

```diff
@@ -52,3 +53,63 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car-list")
+    template_name = "taxi/united_form.html"
```

Comment:
do we really need template_name , fix it all over the code

Reply:
Yes, because I have different template. For example, here is a template that used in both forms.